includes/admin/upgrades/v3/class-data-migrator.php

```diff
@@ -923,11 +955,11 @@ public static function orders( $data = null ) {
 						// Reset any conditional IDs to be safe.
 						$refund_adjustment_id = 0;
 
-						$tax_rate = isset( $meta['_edd_payment_tax_rate'][0] )
-							? (float) $meta['_edd_payment_tax_rate'][0]
-							: 0.00;
```

Comment:
This was causing incorrect tax calculations for Order Item fees. No other use of `get_calculated_tax()` referenced a context-specific tax rate. Removing it falls back to the previously determined rate information.